src/ui/units/wizard/selectors/dialogColor.ts

```diff
@@ -17,6 +17,9 @@ export const selectDialogColorPaletteState = (state: DatalensGlobalState) =>
 export const selectDialogColorGradientState = (state: DatalensGlobalState) =>
     state.wizard.dialogColor.gradientState;
 
+export const selectDialogColorVisualizationId = (state: DatalensGlobalState) =>
+    state.wizard.visualization.visualization?.id;
```

Comment:
It seems that this does not apply directly to the color settings dialog? It is better to put it in a common place

Reply:
Removed this selector and replaced its usage with another existing one